### PR TITLE
docs(storybook): add test runner

### PR DESCRIPTION
**Pull request by Yilun-Sun**
## Description

- add test runner
- fix global layout

## Related Issue

#3298

## Screenshots (if appropriate):

On Storybook site:
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by ckifer** [comment]
oh interesting it looks like we can't build storybook on node less than 16? 🤔 hmm.. Can we adjust this to not run only for Node 14? Not sure how to do that in actions but I'm sure its possible

**Comment by nikolasrieble** [comment]
Wonderful! This unlocks massive value in testing!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -30,3 +30,32 @@ jobs:
           tsc --noEmit
           npm run lint
           npm test
+  build-node-version-16-18:
```
Comment (nikolasrieble) [inline .github/workflows/ci.yml:33]: Given that this step now does more than it did before, how about renaming it? 

> Build node 16 and run storybook
Or such.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -30,3 +30,32 @@ jobs:
...
+        run: npm ci
+      - name: Run build
+        run: npm run build --if-present
+      - name: Test
+        run: |
+          tsc --noEmit
+          npm run lint
+          npm test
+      - name: Serve Storybook and run tests
+        run: npm run test-storybook:ci
```
Comment (nikolasrieble) [inline .github/workflows/ci.yml:60]: Just wondering, why do we not do this in a separate workflow?

**Review by nikolasrieble — APPROVED** [review]
Could you please once verify that the tests also work, by breaking something in the label generation and verifying that the storybook test fails?

**Comment by Yilun-Sun** [comment]
> Could you please once verify that the tests also work, by breaking something in the label generation and verifying that the storybook test fails?

Storybook test fails by set the expected length to a wrong number. (set to 1 for 5-elements data set)
image

**Review by nikolasrieble — APPROVED** [review]
❤️

**Review by ckifer — APPROVED** [review]

**Pull request merged**